releasedir/fs_blobs_dir.go

```diff
@@ -47,6 +48,7 @@ type fsBlobsDirSchema_Blob struct {
 
 	BlobstoreID string `yaml:"object_id,omitempty"`
 	SHA1        string `yaml:"sha"`
+	HREF        string `yaml:"href,omitempty"`
```

Comment:
As this change has been refocused on the notion of validating Blob origin, I would propose we change `HREF` to `URI`, even if we only support HTTP URI's at this point.

This would allow additional URI types to be added in the future.